ksqldb-engine/src/main/java/io/confluent/ksql/function/udf/json/IsJsonString.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021 Confluent Inc.
```

Comment:
The copyright should be updated to 2022.  That may be out of scope for this PR.

Reply:
Fixed in 5452831.